pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -297,10 +297,10 @@ private LoadSheddingStrategy createLoadSheddingStrategy() {
                     Thread.currentThread().getContextClassLoader());
         } catch (Exception e) {
             log.error("Error when trying to create load shedding strategy: {}",
-                    conf.getLoadBalancerLoadPlacementStrategy(), e);
+                    conf.getLoadBalancerLoadSheddingStrategy(), e);
         }
-        log.error("create load shedding strategy failed. using OverloadShedder instead.");
-        return new OverloadShedder();
+        log.error("create load shedding strategy failed. using ThresholdShedder by default.");
+        return new ThresholdShedder();
```

Comment:
why change de default behavior?

Reply:
Because the default value of `loadBalancerLoadSheddingStrategy` is `ThresholdShedder`.

Reply:
@congbobo184 because it is returned only when `loadBalancerLoadSheddingStrategy` is configured with an incorrect value (e.g. a class that does not exist).